### DISCUSSION TITLE
Any exploit for version 121 ChromeOS

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#943] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So I did caub on school wifi but I fogot to do it home wifi so now my Chromebook updated and skivox don't work anymore the alt shift s dosn't work anymore.

**Comment by CGRmags** [comment]
LTMEET flood?

**Reply by CGRmags** [reply]
or LTBEEF flood i forgto

**Reply by AshtonDavies** [reply]
LTMEAT is how it is spelled. LTBEEF is the older one.

**Reply by CGRmags** [reply]
ok LOL i forgot

**Comment by v4pl** [comment]
Ya i tried before but it's not permenit and kinda hard to do

**Reply by AshtonDavies** [reply]
> Ya i tried before but it's not permenit and kinda hard to do

You need more tabs then. I updated the [exploit page](https://chrose.netlify.app/detail/LTMEAT%20Flood) recently, and it should build the right folder for your Chromebook. If it's still not permanent, add more pages.

**Reply by v4pl** [reply]
uh i could be perm? so i just need more pages? but don't i have to wait like 20 minutes for them to load and its perm even after i restart my computer?

**Reply by v4pl** [reply]
and how do you do the auto thing?

**Reply by CGRmags** [reply, marked as answer]
@vaperbest it is permanent but when you restart it, it will always go away, To do the folder put the extension ID in the Background page URL and generate the folder

**Reply by AshtonDavies** [reply]
And don't worry, it doesn't take _that_ long. Probably about 30secs to a minute to respond before you are able to flip the switch.

**Reply by v4pl** [reply]
Um I takes me like 3 minutes because I use 1000 pages.

**Reply by v4pl** [reply]
And you have to flip the switch off before doing it correct?

**Reply by AshtonDavies** [reply]
> And you have to flip the switch off before doing it correct?

Doesn't matter which way it flips. That's a common misconception people keep making that it has to be flipped off. The purpose is to refresh the extension by changing a setting such as that.

**Reply by PenguinLord561** [reply]
When it's the right amount how long should it last?

**Reply by AshtonDavies** [reply]
> When it's the right amount how long should it last?

Indefinitely

**Reply by PenguinLord561** [reply]
I recently found out that they tried to patch this somehow but if you lock the device right before you close the second group of tabs, it still works indefintely

**Reply by AshtonDavies** [reply]
You can also close the second window and open a third time or so on for a longer effect if it doesn't last.

If you can use Dextensify without the crashing problem (clear up resources before you try these), it does it all for you, and if you leave it refreshing the extension long enough, I think you can even close it and it sticks for days at least.

**Reply by Blobby-Boi** [reply]
they should make a dextensify for 10$ chromebooks like mine

**Reply by AshtonDavies** [reply]
It looks like it is set to make 50 iframes at a time. That's lazy, and for that many iframes, it should be based on the Chromebook's RAM level.

**Comment by ohiobombs** [comment]
downgrade to 118
